layers/+lang/latex/packages.el

```diff
@@ -77,10 +77,16 @@
         "mv" 'TeX-view))))
 
 (when (string= latex-build-command "LatexMk")
-(defun latex/init-auctex-latexmk ()
-  (use-package auctex-latexmk
-    :defer t
-    :init (add-hook 'LaTeX-mode-hook 'auctex-latexmk-setup))))
+  (defun latex/init-auctex-latexmk ()
+    (use-package auctex-latexmk
+      :defer t
+      :init
+      (progn
+        (add-hook 'LaTeX-mode-hook
+                  (lambda()
+                    (setq TeX-command-default "LatexMk")))
+        (auctex-latexmk-setup)
```

Comment:
I've addressed the lazy loading issue by wrapping the LatexMk configuration in (eval-after-load "tex-mode". Feels wrong since I guess use-package is designed to avoid needing to do that, but I don't understand use-package well enough. At lease the PR now implements LatexMk support without interfering with lazy loading.